packages/cli/src/commands/generate/helpers.js

```diff
@@ -62,11 +62,26 @@ export const pathName = (path, name) => {
 export const createYargsForComponentGeneration = ({
   componentName,
   filesFn,
+  builder = {
+    force: { type: 'boolean', default: false },
+    typescript: {
+      type: 'boolean',
+      default: false,
+      describe: 'Generate TypeScript files',
+      alias: 'ts',
```

Comment:
👍 alias!

Reply:
💯this should be default setup across generators for sure!